docker-gc

```diff
@@ -275,10 +275,17 @@ $DOCKER images --no-trunc --format "{{.ID}}" --filter dangling=true >> images.al
 > images.reap.tmp
 cat images.all | sort | uniq | while read line
 do
+    # the `docker inspect` command might fail sometimes due to issues like
+    # https://github.com/moby/moby/issues/35747 - don't let that abort this script
+    set +o errexit
     CREATED=$(${DOCKER} inspect -f "{{.Created}}" ${line})
-    ELAPSED=$(elapsed_time $CREATED)
-    if [[ $ELAPSED -gt $GRACE_PERIOD_SECONDS ]]; then
-        echo $line >> images.reap.tmp
+    retval=$?
+    set -o errexit
+    if [ $retval -eq 0 ]; then
+        ELAPSED=$(elapsed_time $CREATED)
+        if [[ $ELAPSED -gt $GRACE_PERIOD_SECONDS ]]; then
+            echo $line >> images.reap.tmp
+        fi
```

Comment:
iiuc, in case the `docker inspect` fails, though the script doesn't abort, it wouldn't GC the docker images right?

Reply:
The image that `docker inspect ...` failed for won't be considered for removal, correct. But my hope is this change allows the script to continue to remove other images that don't return this error.